java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JwtX5tValidator.java

```diff
@@ -51,26 +51,23 @@ public JwtX5tValidator(OAuth2ServiceConfiguration config) {
 	 */
 	@Override
 	public ValidationResult validate(Token token) {
+		if (token == null) {
+			return ValidationResults.createInvalid("No token passed to extract cnf thumbprint");
```

Comment:
No token passed to X5tValidator?

Reply:
jwtx5tvalidator is internal info and already part of the logger name.

Reply:
updated to "No token passed to validate certificate thumbprint"